### ISSUE TITLE
Internal painting

### ISSUE THREAD

**Issue by iteles** [dwyl/home#93]
Feels like the final furlong before we can move in and _dig in_.

Aside from one wall in the digital creations room (https://github.com/dwyl/video/issues/69), the house will get 1 coat of primer (mostly already done everywhere as the kitchen and living room were done this week) and 3 coats of paint.

This is currently scheduled for **Monday 30th September - Wednesday 2nd October 2019**.

### The following will _not_ be painted next week:
+ Top floor bathroom ceilings https://github.com/dwyl/home/issues/72#issuecomment-533666701
+ Top floor corridor ceiling, given we had a window left open in the attic and rain water got into the floor/top corridor ceiling which now needs to dry

Assigned to me to keep an eye on and sort out these exceptions, not to do the painting 😛

**Comment by iteles** [comment]
All walls have now been completed:
![IMG_0278](https://user-images.githubusercontent.com/4185328/68083095-b0c8cd80-fe1c-11e9-9aa3-96b7e387a207.jpg)

Even the chroma key wall:
![IMG_4541](https://user-images.githubusercontent.com/4185328/68083015-c7225980-fe1b-11e9-828f-db2190222f0c.JPG)

But we have had some water damage from a flood:
![IMG_4323 (2)](https://user-images.githubusercontent.com/4185328/68083108-dce44e80-fe1c-11e9-8492-db6b9024cecd.JPG)

Carlos will be rectifying this this week to come.

**Comment by nelsonic** [comment]
@iteles when did you _take_ the photo of the electrician & his apprentice putting up the leisure room light...?

**Comment by iteles** [comment]
It was illustrative for the time being. More photos to be taken next time we're on site.

**Comment by iteles** [comment]
@nelsonic I think that on this, there are definitely some places that need heavy touch-ups such as around the attic staircase which got seriously scuffed during installation. But are there any places where full on painting needs to happen?

From what I can see/remember, I think there are only two places:
+ The leisure room honestly looks like it's a coat of paint short (particularly where we had that small amount of flooding of course but in general, if I look further on the corridor-adjacent wall and the garage-adjacent wall, it looks like it needs another coat
+ The garage
